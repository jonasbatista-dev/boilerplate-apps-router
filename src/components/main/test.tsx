import React from 'react';
import { screen, render } from '@testing-library/react';
import Main from './index';

describe('<main/>', () => {
  it('shold render the heading', () => {
    render(<Main />);
    expect(
      screen.getByRole('heading', {
        name: /Main Component/i,
      }),
    ).toBeInTheDocument();
  });
});
