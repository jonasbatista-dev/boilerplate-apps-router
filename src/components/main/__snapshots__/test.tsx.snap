exports[`<Main/> should render the heading 1`] = `
<main class="sc-Qotzb anddy">
<h1>Main Component</h1>
</main>`;
